planning/behavior_velocity_planner/launch/behavior_velocity_planner.launch.xml

```diff
@@ -50,6 +50,7 @@
     <remap from="~/input/occupancy_grid" to="/sensing/lidar/occupancy_grid"/>
     <remap from="~/output/path" to="path"/>
     <remap from="~/output/stop_reasons" to="/planning/scenario_planning/status/stop_reasons"/>
+    <remap from="~/output/stop_reason2" to="/planning/scenario_planning/status/stop_reason2"/>
```

Comment:
what is this reason2? Typo?

Reply:
Forgot to remove the debug code. Fixed it in https://github.com/autowarefoundation/autoware.universe/pull/1985/commits/6abfdecbd800ba4c820aea92215a45218cfe7eea.